### DISCUSSION TITLE
Technical Questions

### DISCUSSION THREAD

**Discussion by Bcopeland64** [zqfang/GSEApy#115] [Q&A]
Hello All,

I am a first-time user of the gseapy module. I sometimes have technical questions about the usage of the library and troubleshooting code. Is it okay to ask these questions here? Thanks

**Comment by zqfang** [comment]
Yes. Go ahead. But I might rely late. Sorry for that

**Comment by ACDBio** [comment]
Hi! I did a single test with a single custom gene set in GSEAPY prerank and my FWER p-value is lower than FDR q-value and p-value. How can this result be interpreted? Could you describe how is fwer p-value calculated, please? Is it one-sided?

**Reply by zqfang** [reply]
The FWERp and FDRq both use NES null distribution, while p-value uses ES null distribution.

II believe that relying solely on a single test for controlling False Discovery Rate (FDR) or Family-Wise Error Rate (FWER) may not be adequate in your case.

**Reply by ACDBio** [reply]
Could you explain me, what is the difference in FDRq and FWERq calculations in the case with a single gene set, please?
